### PR TITLE
[Mapd] Fix MapD exception for toDateTime

### LINKED ISSUE #1613: Exception  MapD : No match found for function signature toDateTime(<CHARACTER>)
table[table.timestamp_col  < '2000-03-01'].execute()

return error : 
mapd.ttypes.TMapDException: TMapDException(error_msg='Exception: Exception occurred: org.apache.calcite.runtime.CalciteContextException: From line 3, column 21 to line 3, column 46: No match found for function signature toDateTime(<CHARACTER>)')

Backend=MapD  MapD=4.1.1  ibis=0.14  Python=3.6
Ubuntu_18.04  Anaconda_5.1

### PR DESCRIPTION
**Pull request by tonyfast**
This PR tries to resolve #1613 that demonstrates an error when using datetime columns.

### CODE REVIEW

**Comment by kszucs** [comment]
@tonyfast looks like the build is failing

File: ibis/mapd/operations.py [hunk 1]
```diff
@@ -342,7 +342,7 @@ def literal(translator, expr):
                 msg = 'Unsupported subsecond accuracy {}'
                 raise ValueError(msg.format(value))
             value = value.strftime('%Y-%m-%d %H:%M:%S')
-        return "toDateTime('{0!s}')".format(value)
+        return "DATE '{0!s}'".format(value)
```
Comment (cpcloud) [inline ibis/mapd/operations.py]: This should be wrapped in parentheses to make sure the order of operations is what we expect.

**Comment by cpcloud** [comment]
@tonyfast can you rebase?

**Comment by tonyfast** [comment]
Yes I'll have to get to this at the end of the week.  Sorry for letting it sit.

**Comment by xmnlab** [comment]
hey @tonyfast what is the status of this PR?

do you need any help with this PR?

**Comment by kszucs** [comment]
Thanks @tonyfast and @xmnlab!

**Pull request closed without merging**